core/execution/rpc/http_rpc.rs

```diff
@@ -0,0 +1,76 @@
+use std::time::Duration;
```

Comment:
It seems that this is not in the right directory so isn't even being compiled? Should be in src/execution/rpc.